joint_1: 0.0
joint_2: 0.0
joint_3: 0.0
joint_4: 0.0
joint_5: 0.0
joint_6: 0.0
robotiq_2f_85_finger_joint: 0.0
prismatic_joint_rail: 0.0
